sdk/src/record-provider.ts

```diff
@@ -18,64 +13,81 @@ interface RecordSearchParams {
  * implementing this interface.
  */
 interface RecordProvider {
-    account: Account
+    /**
+     * Find encrypted records from the chosen provider
```

Comment:
```suggestion
     * Find encrypted records from the chosen provider.
```

Ensure to put periods on all the comments to conform with the rest of the Provable docs

Reply:
done